### ISSUE TITLE
[Feature Request] Option to use Firebase as storing/sync

### ISSUE THREAD

**Issue by senpl** [TreeNote/TreeNote#47]
I think option to use Firebase as storing/sync would make data exchange between computer and mobile version easier.

**Comment by senpl** [comment]
As I understand it firebase database would make all file synchronizations no longer needed. All data could be stored in firebase database and easly accessed with any interface. Synchronisation on file could go wrong in many ways. Store/sync to database seems easier. In the end user just want their data on mobile and web, not passing theme as file from one app to another.

**Comment by JonasDralle** [comment]
TL;DR
We both loose. It won't happen.

**Comment by JonasDralle** [comment]
Makes sense. Serializing to disk is the cheapest, most efficent, out-of-the-box approach.
I feel like a TreeNote noob just being here due to your Bachelorarbeit and never having used TreeNote before. We have emailed between 2018-2 and 2018-4.

The README does a good job at explaining that this project is "abandoned" (harsh word even tho one could consider it to be a soft abandonment). Idk what the best way is to proceed with this Ticket.
Perhaps flag `deferred indefinitely` or whatever.
